app/helpers/protocols_io_helper.rb

```diff
@@ -146,10 +150,33 @@ def close_open_html_tags(text)
     Nokogiri::HTML::DocumentFragment.parse(text).to_html
   end
 
+  def step_hash_null?(step_json)
```

Comment:
You can refactor `step_hash_null` method like is suggested in [this answer](https://stackoverflow.com/questions/4371716/how-to-avoid-nomethoderror-for-missing-elements-in-nested-hashes-without-repeat).

Reply:
wow, hvala to pa prvič vidim, zgleda zlo uporabno